test/e2e/federated-service.go

```diff
@@ -115,26 +119,27 @@ var _ = framework.KubeDescribe("[Feature:Federation]", func() {
 				}()
 				waitForServiceShardsOrFail(nsName, service, clusters)
 			})
-
-			It("should not be deleted from underlying clusters when it is deleted", func() {
+			It("should be deleted from underlying clusters when OrphanDependents is false", func() {
```

Comment:
This wording doesn't work here, I don't think.  This It() clause falls under the above Describe("Service creation") clause, so the test name will read something weird like "[Feature: Federation] Federated Services - Service creation - should be deleted....". So either the It() clause needs to be prompted to a Describe() clause (e.g. "Service deletion", with It() clauses "with cascading.." etc), or the existing It() clause needs to be reworded appropriately.